### DISCUSSION TITLE
Any way to swap the look of the cursor between insert mode, and normal mode?

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#282] [Q&A]
I am starting to use Meow for, and am struggling with the way the cursor looks in normal mode, and confusing that for being insert mode, and vice versa. Is there any way to swap the look of the cursors?

**Comment by Linerre** [comment]
Perhaps try setting these vars before or right after you `(require 'meow)`?

It seems they haven't been documented in detail at the moment.

https://github.com/meow-edit/meow/blob/6083e746f7c009b16d2178b7a4eddb309f50d738/meow-var.el#L326-L332

**Reply by chookity-pokk** [reply]
So I tried putting that in my config just before `(require 'meow)` and it didn't change anything, so I went and changed the values in `meow-var.el`, and that also didn't change anything. Essentially what I did was just swap out the `'box` for `'(bar . 2)`, and vice versa.

**Reply by DogLooksGood** [reply]
```
(setq meow-cursor-type-insert 'hbar)
```
Try this, put it anywhere you want.

**Reply by chookity-pokk** [reply]
Thank you! I set the normal cursor type to `'hbar`, and the insert cursor to `'box`
